src/test/java/fi/helsinki/cs/tmc/core/communication/UriContains.java

```diff
@@ -0,0 +1,23 @@
+package fi.helsinki.cs.tmc.core.communication;
```

Comment:
Good initiative making this its own class!

I was just wondering if we (c|sh)ould move it to some package like `fi.helsinki.cs.tmc.core.testutils` to distinguish it from test cases. Probably fine to leave it like this for now, and then just move it if we accumulate more such utility classes that are only used in tests.